### PR TITLE
added item function

### PR DESCRIPTION
**Pull request by coquelin77**
added simple item function for getting the singular element from a 1D tensor (analogous to the pytorch command)

### CODE REVIEW

File: heat/core/tests/test_dndarray.py [hunk 1]
```diff
@@ -36,6 +36,11 @@ def test_is_distributed(self):
         data = ht.zeros((4, 4,), split=0)
         self.assertTrue(data.comm.size > 1 and data.is_distributed() or not data.is_distributed())
 
+    def test_item(self):
+        x = ht.zeros((1))
+        self.assertEqual(x.item(), 0)
+        self.assertEqual(type(x.item()), float)
+
```
Comment (ClaudiaComito) [inline heat/core/tests/test_dndarray.py]: test exception?

File: heat/core/dndarray.py [hunk 2]
```diff
@@ -933,6 +933,12 @@ def is_distributed(self):
         """
         return self.split is not None and self.comm.is_distributed()
 
+    def item(self):
+        """
+        Returns the only element of a 1-element tensor. Mirror of the pytorch command by the same name
+        """
```
Comment (ClaudiaComito) [inline heat/core/dndarray.py]: Add example?

File: heat/core/tests/test_dndarray.py (hunk 1 above)
Comment (coquelin77) [reply heat/core/tests/test_dndarray.py]: done

**Comment by coquelin77** [comment]
closes #249

**Pull request merged**